### ISSUE TITLE
Shadowlands Dungeon Support

### ISSUE THREAD

**Issue by MysticalOS** [DeadlyBossMods/DeadlyBossMods#238]
**The Necrotic Wake**
  - [x] Mods created with loading, stats, and combat detection for all bosses
  - [x] Dungeon Boss mods drycoded with warnings/timers/feature
  - [ ] Dungeon boss mod drycodes tested and mods completed from real data
  - [ ] Dungeon Trash mod drycoded with warnings for all notable trash as outlined by Wowhead Guides
  - [ ] Dungeon Trash mod tested thoroughly on Mythic or Mythic+

**Plaguefall**
  - [x] Mods created with loading, stats, and combat detection for all bosses
  - [x] Dungeon Boss mods drycoded with warnings/timers/feature
  - [ ] Dungeon boss mod drycodes tested and mods completed from real data
  - [ ] Dungeon Trash mod drycoded with warnings for all notable trash as outlined by Wowhead Guides
  - [ ] Dungeon Trash mod tested thoroughly on Mythic or Mythic+

**Mists of Tirna Scithe**
  - [x] Mods created with loading, stats, and combat detection for all bosses
  - [x] Dungeon Boss mods drycoded with warnings/timers/feature
  - [ ] Dungeon boss mod drycodes tested and mods completed from real data
  - [ ] Dungeon Trash mod drycoded with warnings for all notable trash as outlined by Wowhead Guides
  - [ ] Dungeon Trash mod tested thoroughly on Mythic or Mythic+

**Halls of Atonement**
  - [x] Mods created with loading, stats, and combat detection for all bosses
  - [x] Dungeon Boss mods drycoded with warnings/timers/feature
  - [ ] Dungeon boss mod drycodes tested and mods completed from real data
  - [ ] Dungeon Trash mod drycoded with warnings for all notable trash as outlined by Wowhead Guides
  - [ ] Dungeon Trash mod tested thoroughly on Mythic or Mythic+

**Spires of Ascension**
  - [x] Mods created with loading, stats, and combat detection for all bosses
  - [x] Dungeon Boss mods drycoded with warnings/timers/feature
  - [ ] Dungeon boss mod drycodes tested and mods completed from real data
  - [ ] Dungeon Trash mod drycoded with warnings for all notable trash as outlined by Wowhead Guides
  - [ ] Dungeon Trash mod tested thoroughly on Mythic or Mythic+

**Theater of Pain**
  - [x] Mods created with loading, stats, and combat detection for all bosses
  - [x] Dungeon Boss mods drycoded with warnings/timers/feature
  - [ ] Dungeon boss mod drycodes tested and mods completed from real data
  - [ ] Dungeon Trash mod drycoded with warnings for all notable trash as outlined by Wowhead Guides
  - [ ] Dungeon Trash mod tested thoroughly on Mythic or Mythic+

**De Other Side**
  - [x] Mods created with loading, stats, and combat detection for all bosses
  - [x] Dungeon Boss mods drycoded with warnings/timers/feature
  - [ ] Dungeon boss mod drycodes tested and mods completed from real data
  - [ ] Dungeon Trash mod drycoded with warnings for all notable trash as outlined by Wowhead Guides
  - [ ] Dungeon Trash mod tested thoroughly on Mythic or Mythic+

**Sanguine Depths**
  - [x] Mods created with loading, stats, and combat detection for all bosses
  - [x] Dungeon Boss mods drycoded with warnings/timers/feature
  - [ ] Dungeon boss mod drycodes tested and mods completed from real data
  - [ ] Dungeon Trash mod drycoded with warnings for all notable trash as outlined by Wowhead Guides
  - [ ] Dungeon Trash mod tested thoroughly on Mythic or Mythic+

**Comment by MysticalOS** [comment]
Additional Tasks:
- [ ] Auto mark mage adds on Amarth in Necrotic Wake dungeon

**Comment by QartemisT** [comment]
Sticking some timer stuff here;

- [ ] Timer ~Slime Injection(Timer329110cd) refreshed before expired. Remaining time is : 2.5. Please report this bug
- [ ] Timer ~Cytotoxic Slash(Timer325552cd) refreshed before expired. Remaining time is : 3.9. Please report this bug

**Comment by MysticalOS** [comment]
This is a tough one. At this point trash and bosse have been thoroughly tested by users, but they are notoriously bad at reporting issues. Lots can still be broken, so until DBM devs or reliable testers have ensured all dungeons have perfect trash and boss mods. This issue remains open.

**Comment by MysticalOS** [comment]
Add warning for Dread Bindings from Grand Overseer from Sanguine Depths

**Comment by Niewa** [comment]
Please add a "switch target" text warning when Oppressive Banner shows up while fighting Xav the Fallen in Theater of Pain

**Comment by MysticalOS** [comment]
It should have one, is it not functional?

**Comment by KyoukanGG** [comment]
Now that Blizzard has returned Xira the Underhanded to the "An Affront of Challengers" (first boss fight) of Theater of Pain, it would be useful to have a say/yell/party spam from the person who is targeted by the Opportunity Strikes stun until they are broken, to aid people with chat bubbles in finding the CC'd person.

**Comment by MysticalOS** [comment]
I thought I already re-enabled that stuff.  let Me double check it

**Comment by MysticalOS** [comment]
Marking all the 9.0 content "complete" at this point. I'm sure issues still remain as users aren't the greatest at reporting them, but at this point I have watched a lot of M+ on twitch and fixed a lot of issues over past few months through 3rd party observation. I'll continue to fix things i see on streams or users report but at this point I'd no longer consider the mods beta or incomplete

9.1 dungeon is a different story. issue remains open for that.

**Comment by MysticalOS** [comment]
https://wago.io/RaidAbilityTimeline replaces that and has been available for many many months, not related to dungeon support.

**Comment by Molkree** [comment]
Some Plagefall timers are off
![image](https://user-images.githubusercontent.com/12860284/149527544-63b68083-700c-4efe-bf2f-19cb9edb02c4.png)
![image](https://user-images.githubusercontent.com/12860284/149527554-6df88839-b11e-4c66-8689-1270ad9819e8.png)
[Log](https://gist.github.com/Molkree/c506e8f92828e5bfa4ac245f68a40a15)

**Comment by awcarr** [comment]
Any chance we can get Drifting Star from Adorned Starseer in Tazavesh added to the audible frontal abilities?

**Comment by MysticalOS** [comment]
Probably eventually. this dungeon still effectively only supports one boss because mega dungeons are always hard to support when I don't have reliable testers/loggers for this sort of thing and pugging this dungeon has and always will be a nightmare on Mythic or higher

i did plan to queue heroic versions eventually to try and get some data for them, but only after raid stuff settles down.

**Comment by MysticalOS** [comment]
Also, since I have no idea if those reports from molk are current, i'm just gonna flag them as outdated and only take new reports for those dungeons.

**Comment by MysticalOS** [comment]
Issue now low priority, it's unlikely i'll be able to test this dungeon on hard mode for foreseeable future (probably not for 2-3 expansions at this point).

Ticket remains open as low priority now. Mods were adequate enough for M+ but actually making sure they work on hard mode is just not something that can be done right now.
